inspector-apm/inspector-symfony/1.0/config/packages/inspector.yaml

```diff
@@ -0,0 +1,56 @@
+inspector:
+    ingestion_key: '%env(INSPECTOR_INGESTION_KEY)%'
+    url: https://ingest.inspector.dev
+    enabled: true
+    unhandled_exceptions: true
+    query: true
+    query_bindings: false
+    templates: true
+    user: true
+    transport: async
+    server_sampling_ratio: 0
```

Comment:
Do we really need all these default values? 

I would prefer a link to the bundles documentation instead.

Reply:
Hi, I removed the not mandatory config parameters.